frame/dapps-staking/src/pallet/mod.rs

```diff
@@ -22,6 +22,7 @@ use sp_std::convert::From;
 const STAKING_ID: LockIdentifier = *b"dapstake";
 
 #[frame_support::pallet]
+#[allow(clippy::module_inception)]
```

Comment:
Imo, we should fix this.
It's been a thorn in my eye for quite some time.

But to keep this task focused, can you crate a new backlog item for this?
It should be low prio, P5.

Reply:
It's easy to fix this, by just changing directory name or pub mod name.
but what it could be.